src/nevermined/Provider.ts

```diff
@@ -3,18 +3,13 @@ import { Instantiable, InstantiableConfig } from '../Instantiable.abstract'
 import { TxParameters } from '../keeper/contracts/ContractBase'
 
 /**
- * Pro submodule of Nevermined.
+ * Providers API that allows the basic management of the provider accounts associated to an asset.
  */
-export class Provider extends Instantiable {
-    /**
-     * Returns the instance of Provider.
-     * @returns {@link Provider}
-     */
-    public static async getInstance(config: InstantiableConfig): Promise<Provider> {
-        const instance = new Provider()
-        instance.setInstanceConfig(config)
+export class Providers extends Instantiable {
```

Comment:
Is this used anywhere?

Reply:
The functionality is necessary but not sure if it's being used. Managing the providers of an asset (add, revoke) needs to be exposed.